BUILD.md

```diff
@@ -422,6 +422,27 @@ proxy binary:
 bin/docker-build-proxy
 ```
 
+#### Locally built proxy
+
+If you want to deploy a locally built proxy, you can build it in the [`linkerd2-proxy`](https://github.com/linkerd/linkerd2-proxy) repo by running:
+
+```bash
+DOCKER_TAG=cr.l5d.io/linkerd/proxy:dev make docker
+```
+
+Then, in this repo, run:
+
+```bash
+./bin/k3d image import cr.l5d.io/linkerd/proxy:dev
+```
+
+Now, to make a pod use your image, add the following annotations to it:
+
+```yaml
+config.linkerd.io/proxy-version: dev
+config.linkerd.io/proxy-await: disabled
```

Comment:
I think we can avoid this by updating the devel docker container to use a more recent base image
```suggestion
```

@kleimkuhler will submit a proxy PR

Reply:
This was just fixed by linkerd/linkerd2-proxy#999 so all builds rebased on `main` (d87d0dba8afb0912073e9d3b767c252ac7774f88) will have the `linkerd-await` wrapper.